packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -45,11 +46,12 @@ const useUtilityClasses = (ownerState) => {
       expanded && 'expanded',
       focused && 'focused',
       fullWidth && 'fullWidth',
+      multiple && 'multiple',
```

Comment:
I think we should keep the `multiple` class only on the `inputRoot` slot and remove for others (`root` and `input`). We need it only for the flex-wrap on the input root.